components/speedreader/rust/ffi/config.gni

```diff
@@ -0,0 +1,75 @@
+declare_args() {
+  use_speedreader_shared_lib = is_mac
+  speedreader_flags = ""
+}
+
+speedreader_shared_lib_install_name = ""
```

Comment:
this stuff was never meant to be duplicated in other repos, in a follow up we need to reorganize this stuff

Reply:
reorganised some if it here, still need to resolve a linking issue on linux, can you take another look?